src/control/server/mgmt_iosrv.go

```diff
@@ -40,25 +40,22 @@ func (c *controlService) callDrpcMethodWithMessage(
 
 	drpcResp, err := makeDrpcCall(c.drpc, mgmtModuleID, methodID, body)
 	if err != nil {
-		fmt.Printf("%+v\n", err)
 		return nil, errors.WithStack(err)
 	}
 
-	fmt.Printf("%+v\n", drpcResp)
+	// unmarshal daos response message returned in drpc response body
 	resp = &pb.DaosResponse{}
-	// TODO: unmarsal daos response message returned in drpc response body,
-	//       populated by io server mgmt drpc handler
-	//	err = proto.Unmarshal(drpcResp.Body, resp)
-	//	if err != nil {
-	//		return nil, fmt.Errorf("invalid dRPC response body: %v", err)
-	//	}
+	err = proto.Unmarshal(drpcResp.Body, resp)
+	if err != nil {
+		return nil, fmt.Errorf("invalid dRPC response body: %v", err)
```

Comment:
This is good for now, I am going to propose a drpc error message type in the future we can use here.